vite.config.js

```diff
@@ -33,10 +33,8 @@ export default defineConfig(({ command, mode }) => {
 
   const warehouseProxyServer = {
     target: env.HMIS_SERVER_URL || DEFAULT_WAREHOUSE_SERVER,
+    xfwd: true,
     changeOrigin: true, // sets Host header
-    headers: {
-      Origin: env.HMIS_SERVER_URL || DEFAULT_WAREHOUSE_SERVER,
-    },
```

Comment:
see https://github.com/greenriver/hmis-frontend/blob/release-125/docker/nginx.conf and template files in the same dir